idaes/generic_models/properties/core/eos/ceos.py

```diff
@@ -772,6 +857,40 @@ def gibbs_mol_phase_comp(b, p, j):
                 b.entr_mol_phase_comp[p, j] *
                 b.temperature)
 
+    @staticmethod
+    def heat_capacity_ratio_phase(blk, p):
+        pobj = blk.params.get_phase(p)
+        if not (pobj.is_vapor_phase() or pobj.is_liquid_phase()):
```

Comment:
Do we need these checks at all? I don't think any of the other methods in the ceos module use them.

Reply:
I followed the structure used in the enthalpy method. It had the checks, so I put them here as well. I have now removed them.